packages/utils/src/combineEventHandlers.test.js

```diff
@@ -1,45 +1,69 @@
-import combineEventHandlers from "./combineEventHandlers";
+import combineEventHandlers, {
+  memoizeCombineEventHandlers
+} from "./combineEventHandlers";
 
 describe("utils/combineEventHandlers", () => {
-  it("returns an event handler", () => {
-    expect(combineEventHandlers()).toEqual(expect.any(Function));
-  });
+  describe("combineEventHandlers", () => {
+    it("returns an event handler", () => {
+      expect(combineEventHandlers()).toEqual(expect.any(Function));
+    });
 
-  describe("combined event handler", () => {
-    const args = [{}, 1, "2", true];
-    const handler1 = jest.fn();
-    const handler2 = jest.fn();
+    describe("combined event handler", () => {
+      const args = [{}, 1, "2", true];
+      const handler1 = jest.fn();
+      const handler2 = jest.fn();
 
-    beforeEach(() => {
-      jest.resetAllMocks();
-    });
+      beforeEach(() => {
+        jest.resetAllMocks();
+      });
 
-    it("calls each handler with the given arguments", () => {
-      const result = combineEventHandlers(handler1, handler2);
+      it("calls each handler with the given arguments", () => {
+        const result = combineEventHandlers(handler1, handler2);
 
-      result(...args);
+        result(...args);
 
-      expect(handler1).toBeCalledWith(...args);
-      expect(handler2).toBeCalledWith(...args);
-      expect(handler1).toHaveBeenCalledTimes(1);
-      expect(handler2).toHaveBeenCalledTimes(1);
-    });
+        expect(handler1).toBeCalledWith(...args);
+        expect(handler2).toBeCalledWith(...args);
+        expect(handler1).toHaveBeenCalledTimes(1);
+        expect(handler2).toHaveBeenCalledTimes(1);
+      });
 
-    it("ignores falsy values for handlers", () => {
-      const result = combineEventHandlers(
-        undefined,
-        handler1,
-        null,
-        handler2,
-        false
-      );
+      it("ignores falsy values for handlers", () => {
+        const result = combineEventHandlers(
+          undefined,
+          handler1,
+          null,
+          handler2,
+          false
+        );
+
+        result(...args);
 
-      result(...args);
+        expect(handler1).toBeCalledWith(...args);
+        expect(handler2).toBeCalledWith(...args);
+        expect(handler1).toHaveBeenCalledTimes(1);
+        expect(handler2).toHaveBeenCalledTimes(1);
+      });
+    });
+  });
+
+  describe("memoizeCombineEventHandlers", () => {
+    it("memoizes combineEventHandlers", () => {
+      const createHandlers = memoizeCombineEventHandlers();
+      const handler1 = () => {};
+      const handler2 = () => {};
+      const handler3 = () => {};
+      const handler4 = () => {};
 
-      expect(handler1).toBeCalledWith(...args);
-      expect(handler2).toBeCalledWith(...args);
-      expect(handler1).toHaveBeenCalledTimes(1);
-      expect(handler2).toHaveBeenCalledTimes(1);
+      expect(createHandlers(handler1, handler2)).toEqual(
+        createHandlers(handler1, handler2)
```

Comment:
Consider using `toHaveBeenCalled` on `CombineEventHandlers` here to test memoization